### PR TITLE
Add batch of resources

### PR DESCRIPTION
**Pull request by NielsRogge**
# What does this PR do?

This PR adds a batch of resources, primilarly for all image classifiers.

### CODE REVIEW

File: docs/source/en/model_doc/dit.mdx [hunk 1]
```diff
@@ -64,4 +64,14 @@ A notebook that illustrates inference for document image classification can be f
...
\ No newline at end of file
+This model was contributed by [nielsr](https://huggingface.co/nielsr). The original code can be found [here](https://github.com/microsoft/unilm/tree/master/dit).
+
+## Resources
+
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with DiT.
+
+<PipelineTag pipeline="image-classification"/>
+
+- [`BeitForImageClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/image-classification) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/image_classification.ipynb).
```
Comment (sgugger) [inline docs/source/en/model_doc/dit.mdx:75]: This is the Dit model here.

**Review by sgugger — APPROVED** [review]
Thanks for adding those!
One of your changes broke the documentation build it appears though.

**Review by stevhliu — APPROVED** [review]
Wow thanks for adding all these! 👍

File: docs/source/en/model_doc/dit.mdx (hunk 1 above)
Comment (NielsRogge) [reply docs/source/en/model_doc/dit.mdx:75]: There's no Dit model, it's equivalent to Beit ;)

**Comment by NielsRogge** [comment]
Thanks for your review! It's unclear to me why the "build PR documentation" check is failing, thought it had to do with the pipeline tags, but it's still failing. Any insight would be greatly appreciated

**Comment by sgugger** [comment]
You will have to isolate which file triggers the issue and then which line inside that file by trial and error I'm afraid. That's one of the reason smaller PRs are easier to deal with :-)

File: docs/source/en/model_doc/deformable_detr.mdx [hunk 2]
```diff
@@ -24,7 +24,7 @@ The abstract from the paper is the following:
 Tips:
 
 - One can use [`DeformableDetrImageProcessor`] to prepare images (and optional targets) for the model.
-- Training Deformable DETR is equivalent to training the original [DETR](detr) model. Demo notebooks can be found [here](https://github.com/NielsRogge/Transformers-Tutorials/tree/master/DETR).
+- Training Deformable DETR is equivalent to training the original [DETR](detr) model. See the [resources](resources) section below for demo notebooks.
```
Comment (MKhalusova) [inline docs/source/en/model_doc/deformable_detr.mdx:27]:
```suggestion
- Training Deformable DETR is equivalent to training the original [DETR](detr) model. See the [resources](#resources) section below for demo notebooks.
```

File: docs/source/en/model_doc/dpt.mdx [hunk 3]
```diff
@@ -28,37 +28,42 @@ alt="drawing" width="600"/>
 
 This model was contributed by [nielsr](https://huggingface.co/nielsr). The original code can be found [here](https://github.com/isl-org/DPT).
 
+## Resources
+
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with DPT.
+
+<PipelineTag pipeline="depth-estimation"/>
```
Comment (MKhalusova) [inline docs/source/en/model_doc/dpt.mdx:35]: This is what likely causes the build failure. The depth estimation pipeline. 
When trying to build this page locally I'm getting :
500
Cannot read properties of undefined (reading 'name')
TypeError: Cannot read properties of undefined (reading 'name')
    at PipelineTag.svelte:12:31
    at Object.$$render (/private/var/folders/jb/59bqr82j2mx1qv1zqbt4ntn80000gn/T/tmpuwohzafj/kit/node_modules/svelte/internal/index.js:1745:22)
    at eval (/src/routes/model_doc/glpn.mdx:126:91)
    at Object.$$render (/private/var/folders/jb/59bqr82j2mx1qv1zqbt4ntn80000gn/T/tmpuwohzafj/kit/node_modules/svelte/internal/index.js:1745:22)
    at Object.default (root.svelte:43:47)
    at eval (/src/routes/__layout.svelte:40:157)
    at Object.$$render (/private/var/folders/jb/59bqr82j2mx1qv1zqbt4ntn80000gn/T/tmpuwohzafj/kit/node_modules/svelte/internal/index.js:1745:22)
    at root.svelte:37:45
    at $$render (/private/var/folders/jb/59bqr82j2mx1qv1zqbt4ntn80000gn/T/tmpuwohzafj/kit/node_modules/svelte/internal/index.js:1745:22)
    at Object.render (/private/var/folders/jb/59bqr82j2mx1qv1zqbt4ntn80000gn/T/tmpuwohzafj/kit/node_modules/svelte/internal/index.js:1753:26)

Removing the pipeline tag solves the issue.

File: docs/source/en/model_doc/glpn.mdx [hunk 4]
```diff
@@ -41,6 +40,14 @@ alt="drawing" width="600"/>
 
 This model was contributed by [nielsr](https://huggingface.co/nielsr). The original code can be found [here](https://github.com/vinvino02/GLPDepth).
 
+## Resources
+
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with GLPN.
+
+<PipelineTag pipeline="depth-estimation"/>
```
Comment (MKhalusova) [inline docs/source/en/model_doc/glpn.mdx:47]: Here, same as above. The "depth estimation" pipeline is the likely culprit"

**Review by MKhalusova — APPROVED** [review]

File: docs/source/en/model_doc/segformer.mdx [hunk 5]
```diff
@@ -84,6 +84,22 @@ Tips:
...
 
+## Resources
+
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with SegFormer.
+
+<PipelineTag pipeline="image-classification"/>
+
+- [`SegformerForImageClassification`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/image-classification) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/image_classification.ipynb).
+
+<PipelineTag pipeline="semantic-segmentation"/>
```
Comment (MKhalusova) [inline docs/source/en/model_doc/segformer.mdx:95]: Another tag that is causing the doc build failure. 
```suggestion
```

**Pull request merged**